packages/visualizations-react/stories/Table/Table.stories.tsx

```diff
@@ -76,4 +81,10 @@ export const Unstyled = Template.bind({});
 Unstyled.args = {
     data,
     options: { ...options, unstyled: true },
+};
+
+export const Loading = Template.bind({});
+Loading.args = {
+    data: fetchingData,
+    options,
```

Comment:
A remark: how many 'loading rows' should we display? On which prop we can use to get this number ? 

Is very common to have an empty `data` array when `loading` is true

Reply:
Skeleton must be displayed in each cell of the table. `data` can be an empty array, it is `isLoading` which conditions the display of skeletons.

Reply:
Now we add as many lines as the `recordsPerPage` if it exists, otherwise we add 5 lines.